packages/eslint-plugin/src/rules/object-curly-spacing.ts

```diff
@@ -24,10 +24,12 @@ export default createRule<Options, MessageIds>({
   // eslint-disable-next-line eslint-plugin/prefer-message-ids,eslint-plugin/require-meta-type,eslint-plugin/require-meta-schema,eslint-plugin/require-meta-fixable -- all in base rule - https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/issues/274
   meta: {
     ...baseRule.meta,
+    deprecated: true,
     docs: {
       description: 'Enforce consistent spacing inside braces',
       extendsBaseRule: true,
     },
+    replacedBy: ['@stylistic/object-curly-spacing'],
```

Comment:
OCD eye twitch: all the other ones you added on consecutive lines

Reply:
Heh, the other ones are placed before a `type:`. So they somewhat don't make the sorting less alphabetical. This one doesn't have a `type:`...